modules/core/src/capture/video-capture.js

```diff
@@ -159,6 +159,7 @@ export class VideoCapture {
     if (this.isRecording()) {
       console.log(`Stopping recording.  Recorded for ${this.durationMs}ms.`);
       this.recording = false;
+      this.capturing = false;
```

Comment:
Want to highlight that this fixes an intermittent bug I'd been puzzled by. Thanks for identifying it :)